package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bip39",
   "version": "2.5.0",
   "description": "Bitcoin BIP39: Mnemonic code for generating deterministic keys",
-  "main": "index.js",
+  "main": "src/index.js",
```

Comment:
The moving of `index.js` should not be a breaking change (if someone is just doing a `require('bip39')`